godot-core/src/registry/callbacks.rs

```diff
@@ -186,21 +186,24 @@ pub unsafe extern "C" fn default_get_virtual<T: UserClass>(
     T::__default_virtual_call(method_name.as_str())
 }
 
+/// # Safety
+///
+/// - `r_is_valid` must be assignable.
+/// - `r_is_valid` comes uninitialized and must be set.
 pub unsafe extern "C" fn to_string<T: cap::GodotToString>(
```

Comment:
Safety sections are the requirements a caller must fulfill, not the requirements on the function implementation (2nd bullet point).

But let's remove it altogether, as only Godot calls it and none of the other callbacks has it. You can still mention the fact that  
> `is_valid` comes uninitialized and must be set by this function

as a regular (non-safety) comment.